.changeset/afraid-buckets-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-popover": minor
```

Comment:
**suggestion(non-blocking):** Could you please change this to `patch` as you are fixing an issue with the existing implementation?

To clarify a bit more, the public API of Popover is not changing as `aria-label` existed already as a prop, but instead the internal implementation is being fixed by applying the correct prop/attribute to the `PopoverDialog` element.